ci-operator/step-registry/assisted/baremetal/operator/ztp/assisted-baremetal-operator-ztp-workflow.yaml

```diff
@@ -2,7 +2,6 @@ workflow:
   as: assisted-baremetal-operator-ztp
   steps:
     allow_best_effort_post_steps: true
-    allow_skip_on_success: true
```

Comment:
Be aware, enabling gather on successful ZTP jobs is potentially going to cause timeouts for successful jobs. At least in the past we were running on a very short buffer (like 5 minutes till the timeout). It's a good change, but it should be monitored. If we see too many timeouts suddenly, we'll need to think about some smart solution

Reply:
ah, you mean `post` steps may timeout and make successful jobs fail? Right, this is bad, lets keep it WIP until we figure out a good solution for this

Reply:
Yes, exactly this scenario... The real meat of the job succeeds, then gather in post times out and the whole job fails as "timeout". Still, this was in the past, maybe this problem now disappeared as we use more modern CPUs, as the universe is in a different position and all sorts of different reasons. By no means I want to stop PR by this, just something we need to pay attention at

Reply:
Fixed - not touching allow_skip_on_success